duo_api.gemspec

```
@@ -1,6 +1,6 @@
 Gem::Specification.new do |s|
   s.name        = 'duo_api'
-  s.version     = '1.0.1'
+  s.version     = '1.0.2'
```

```suggestion
  s.version     = '1.1.0'
```
Can you bump the minor version since this is more of a new feature than a bug fix?